source/package.json

```diff
@@ -28,18 +28,29 @@
     "grunt-ng-annotate": "^3.0.0",
     "grunt-ngdocs": "^0.2.11",
     "grunt-plato": "^1.4.0",
-    "jest": "^23.6.0"
+    "http-server": "^0.11.1",
```

Comment:
Do we still need `http-server` if we are using express?

Reply:
Will remove as well